private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/cmd/Main.java

```diff
@@ -256,21 +259,31 @@ private static void writeLockFile(
       EventListener listener,
       ResolverConfig config,
       ResolutionRequest request,
-      Set<DependencyInfo> infos)
+      Set<DependencyInfo> infos,
+      Set<Conflict> conflicts)
       throws IOException {
     listener.onEvent(new PhaseEvent("Building lock file"));
     Path output = config.getOutput();
 
     listener.close();
 
+    Function<Coordinates, String> simpleForm =
+        c -> c.getGroupId() + ":" + c.getArtifactId() + ":" + c.getVersion();
```

Comment:
would it make sense to implement `Coordinates#toString`  instead?

Reply:
This format is what we had before with the `coursier`-based resolver, and my assumption is that people have built tooling around that specific format. If we don't mind, then I can definitely use `toString` as it's less code for us :)

Reply:
Switched to `toString`. People can file an issue if they've built a workflow around an undocumented part of the lock file.